config/jest.config.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
     path.resolve('config/jest/setupEnzyme.js'),
     path.resolve('config/jest/setupGlobals.js'),
   ],
-  setupTestFrameworkScriptFile: path.resolve('node_modules/jest-enzyme/lib/index.js'),
+  setupFilesAfterEnv: [
+    path.resolve('node_modules/jest-enzyme/lib/index.js'),
+    path.resolve('config/jest/setupStyledComponents.js'),
```

Comment:
To avoid the unnecessary file, could we change this to its node_modules path?
```diff
- path.resolve('config/jest/setupStyledComponents.js'),
+`path.resolve('node_modules/jest-styled-components/src/index.js')`
```

Alternatively, can we link to a single jest config file and have all our needed `import` statements in there?